CHANGELOG.md

```
@@ -3110,7 +3185,8 @@ OTLP tracing, OAuth for remote MCP), the MCP stdio bridge, and the
 offline tooling (`import`, `extract`, `inspect`, `estimate`).
 Published to crates.io and GHCR.
 
-[Unreleased]: https://github.com/t0k0sh1/taguru/compare/v0.9.1...HEAD
+[Unreleased]: https://github.com/t0k0sh1/taguru/compare/v0.9.2...HEAD
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)